src/components/Request/FormFields/AdvertisementLength.tsx

```diff
@@ -1,19 +1,20 @@
-import { Radio } from "@fluentui/react-components";
 import "components/Request/Request.css";
 import { RHFRequest } from "components/Request/NewRequestForm";
-import BACRadioGroup from "components/BaseFormFields/BACRadioGroup";
+import BACInput from "components/BaseFormFields/BACInput";
 
 const AdvertisementLength = () => {
   return (
     <div className="requestFieldContainer">
-      <BACRadioGroup<RHFRequest>
+      <BACInput<RHFRequest>
         name="advertisementLength"
         labelText="Advertisement Length"
-        fieldProps={{ layout: "horizontal" }}
-      >
-        <Radio value="Normal" label="Normal Period" />
-        <Radio value="Extended" label="Extended Period (> 14 days)" />
-      </BACRadioGroup>
+        fieldProps={{
+          type: "number",
+          step: "1",
+          min: "7",
+          max: "30",
```

Comment:
These aren't having any impact, as React Hook Form is doing validations -- and since there isn't a rule on the field for min/max it is allowing any value